modules/customize-certificates-rotate-service-serving.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/service-serving-certificate.adoc
+
+[id="rotate-service-serving-{context}"]
+= Manually rotate the generated service certificate
+
+.Prerequisites
+
+* A secret containing the certificate and key pair must 
+have been generated for the service.
+
+.Procedure
+
+. Delete the generated secret for the service:
++
+----
+$ oc delete secret <service.name>
```

Comment:
This needs an annotation.

I'd also add the step to determine what the right secret is, if possible.

Reply:
I've included an annotation and the steps for determining the secret.